projects/archiver/src/tasks/zip/helpers/zipper.ts

```diff
@@ -39,19 +49,29 @@ export const zip = async (
                 ? file.replace(`${options.removeFromOutputPath}`, '')
                 : file
 
-            // Remove the image size from the path and replace with a static `images`
-            // It is currently needed for server side rendering html bundle
-            const zipPath = options.replaceImageSize
-                ? path.replace(`/${options.replaceImageSize}/`, '/images/')
-                : path
-
             console.log(`getting ${file}`)
             const s3response = await s3
                 .getObject({ Bucket: bucket.name, Key: file })
                 .promise()
             if (s3response.Body == null) return false
 
-            archive.append(s3response.Body as Buffer, {
+            // Remove the 'image size' from the path and replace with a static `images`
+            // to support new SSR compliant clients. Example: from 'media/phone/img1.png' to 'media/images/img1.png'
```

Comment:
Will this mean these _from the web_ images won’t be sized depending on the device they are viewed on? This could be a nice enhancement, if so.

Reply:
Oh good spot @paperboyo - this means we'll serve whateever size of images apps rendering gives us - which will have to be the tabletXL size - so everyone will end up with bigger images. Unless apps rendering provides a picture element (which it's presumably already doing for the live apps) and we have a more complicated regex to replace that with a downloaded image.

Reply:
> Unless apps rendering provides a picture element

@JamieB-gu would know best, but I think currently they have multiple sources in an `img`, but will switch to full `picture` later? Not sure how safe would be to try to be (too?) clever with the regex, though…

Reply:
Apps-rendering uses image-rendering, which means that it outputs the `<picture>` element seen [here](https://github.com/guardian/image-rendering/blob/d24dbd17432f79040f01c48da87e42931ab28c32/src/components/img.tsx#L62). Regexing URLs in this way will update all the URLs in the `<source>` and `<img>` tags.

Reply:
Yeah, yeah, my comment was just about making these fastly fallbacks different for different devices. And I was wrong about the `img`, thanks!

Reply:
@paperboyo to answer to your original question, 
short ans: Yes, the device will still choose the best image size
long ans: we now have a new image bundle for the rendered article which has a slightly simplified folder structure without the `image size` in the tree hierarchy. The device will download the appropriate image bundle in the first place and this change will not impact anyway on the image quality.

Reply:
I'm going to merge this PR on top of another PR - so there will still chance for refactoring if we want

Reply:
@mohammad-haque Thank you, that’s great to hear!